nucypher/policy/policies.py

```diff
@@ -115,8 +104,24 @@ def block_until_success_is_reasonably_likely(self):
         # `percent_to_complete_before_release` successes. For now just letting it fire.
         self._worker_pool.block_until_target_successes()
         completed = self.completed
-        self.log.debug(f"The minimal amount of nodes ({len(completed)}) was contacted "
+        self.log.debug(f"The minimal amount of nodes ({len(completed)}) were contacted "
                        "while blocking for treasure map publication.")
+
+        successes = self._worker_pool.get_successes()
+        responses = {ursula.checksum_address: status for ursula, status in successes.items()}
+        if not all(response.status_code == 201 for response in responses.values()):
+            report = "\n".join(f"{address}: {status}" for address, status in responses.items())
+            self.log.debug(f"Policy enactment failed. Request statuses:\n{report}")
+
+            # OK, let's check: if any Ursulas claimed we didn't pay,
+            # we need to re-evaluate our situation here.
+            claims_of_freeloading = sum(response.status_code == 402 for response in responses.values())
```

Comment:
Small thing, but, given how it is used, could be just `any` instead of `sum`

Reply:
✔️